packages/app/src/app/components/dashboard/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+import { Button as ButtonComponent } from '@codesandbox/components';
+
+export const Button = styled(ButtonComponent)`
```

Comment:
Is this actually a variant of the Button component or just some style adjustments to fit a specific scenario in the dashboard? I would either move this to the Button component if it will be a common use case (as a variant?) or I would call it `StyledButton` to make it easier when moving the screens later